graylog2-web-interface/src/components/users/ShowConfigValue.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+import * as React from 'react';
+
+import { Row, Col } from 'components/graylog';
+import { TextLabel } from 'components/common';
+
+type Props = {
+  label: string,
+  value: React.Node,
+};
+
+const ShowConfigValue = ({ label, value }: Props) => (
+  <Row>
+    <Col xs={3}>
+      <TextLabel>{label}</TextLabel>
+    </Col>
+    <Col xs={9}>
+      {value ?? '-'}
+    </Col>
+  </Row>
+);
+
+export default ShowConfigValue;
```

Comment:
@kmerz I just realized I've implemented `ShowConfigValue` and `FormFieldRead`, which do basically the same. I will update this in a follow-up PR as well.